packages/ui/react-ui/src/types.ts

```diff
@@ -0,0 +1,54 @@
+import { CSSProperties, MouseEvent } from "react";
+
+export enum clientNames {
+    reactUi = "client-sdk-react-ui",
+}
+
+export enum baseUrls {
+    prod = "https://www.crossmint.io",
+    dev = "http://localhost:3001",
+}
+
+export enum customHeaders {
+    clientVersion = "X-Client-Version",
+    clientName = "X-Client-Name",
+}
+
+enum theme {
+    light = "light",
+    dark = "dark",
+}
+
+export enum mintingContractTypes {
+    CANDY_MACHINE = "candy-machine",
+    SOLANA_AUCTION = "solana-auction",
+    ERC_721 = "erc-721",
+}
+
+export interface PayButtonConfig {
```

Comment:
On discord we decided to not have any client side validation / enforcement of what goes on the mintConfig property, so we can release new contracts without doing a new SDK release each time

As a result,, this type is not needed and should be replaced with an "any"